SignalServiceKit/src/Devices/OWSReadReceiptsForLinkedDevicesMessage.h

```diff
@@ -1,15 +1,17 @@
-//  Copyright © 2016 Open Whisper Systems. All rights reserved.
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
 
 #import "OWSOutgoingSyncMessage.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 @class OWSReadReceipt;
 
-@interface OWSReadReceiptsMessage : OWSOutgoingSyncMessage
+@interface OWSReadReceiptsForLinkedDevicesMessage : OWSOutgoingSyncMessage
```

Comment:
Renaming to distinguish between "linked device" read receipts and "sender/recipient" read receipts.

Reply:
It would be nice to converge with Android/Desktop's nomenclature... once we figure out what that is.

Reply:
+1